internal/engine/filter/filter.go

```diff
@@ -12,7 +13,7 @@ import (
 // Filters is a list of filters created in Gmail.
 type Filters []Filter
 
-func (fs Filters) String() string {
+func (fs Filters) DebugString(extraInfo bool) string {
```

Comment:
I would prefer to keep the `String` function as is, since it's a standard function used by `fmt`: https://pkg.go.dev/fmt#Stringer. Could you add a `DebugString()` function instead?